bootstrap.js

```diff
@@ -5,6 +5,12 @@ module.exports = function(grunt) {
     config: config
   });
 
+  // Allow Gruntconfig.json to set default descriptions.
+  if (config.help) {
+    grunt.config('help', config.help);
+    grunt.config('config.help', '');
```

Comment:
If there's no functional value in removing the setting in 'config.help', then let's leave it there.